components/financial_assistance/app/views/financial_assistance/benefits/_insurance_kinds.html.erb

```diff
@@ -43,7 +43,13 @@
       <% end %>
     </div>
     <div class="col-md-12 add-more-link-<%=insurance_kind%>" id="add-more-link">
-      <%= link_to "<span class='fa-icon fa-stack plus-mr' aria-hidden='true'><i class= 'fa fa-circle-thin fa-stack-2x'></i> <i class='fa fa-plus fa-stack-1x'></i></span> <b>Add More</b> #{l10n("faa.question.#{insurance_kind}", short_name: EnrollRegistry[:enroll_app].setting(:short_name).item)}".html_safe, "javascript:void(0);", :id => "add_new_insurance_kind", :class => "#{'hidden' unless @applicant.benefits.where(kind: kind).of_insurance_kind(insurance_kind).present? }" %>
+      <a tabindex="0" onkeydown="handleButtonKeyDown(event, 'add_new_insurance_kind')" id="add_new_insurance_kind" class="<%= 'hidden' unless @applicant.benefits.where(kind: kind).of_insurance_kind(insurance_kind).present? %>">
+        <span class="fa-icon fa-stack plus-mr" aria-hidden="true">
+          <i class="fa fa-circle-thin fa-stack-2x"></i>
+          <i class="fa fa-plus fa-stack-1x"></i>
+        </span>
+        <b> <%= l10n("add_more") %> </b> <%= l10n("faa.question.#{insurance_kind}", short_name: EnrollRegistry[:enroll_app].setting(:short_name).item) %>
+      </a>    
```

Comment:
improved the add more button logic in several places to make it more readable and pass codesweep